src/main/dynamo/Beyonce.ts

```diff
@@ -41,12 +42,14 @@ export interface ScanOptions {
  * does auto mapping between JSON <=> DynamoDB Items
  */
 export class Beyonce {
-  private client: DynamoDB.DocumentClient
+  private client: DynamoDBDocumentClient
   private jayz?: JayZ
   private consistentReads: boolean
 
   constructor(private table: Table<string, string>, dynamo: DynamoDB, options: Options = {}) {
-    this.client = new DynamoDB.DocumentClient({ service: dynamo })
+    this.client = DynamoDBDocumentClient.from(dynamo, {
+      marshallOptions: { removeUndefinedValues: true }
```

Comment:
Attempting to write keys with undefined values results in errors unless specifying this option.